src/Core/Framework/Api/Controller/AuthController.php

```diff
@@ -60,4 +63,14 @@ public function token(Request $request): Response
 
         return (new HttpFoundationFactory())->createResponse($response);
     }
+
+    #[Route(path: '/api/oauth/sso/config', name: 'api.oauth.sso.url', defaults: ['auth_required' => false], methods: ['GET'])]
+    public function loginButtonConfig(Request $request): JsonResponse
+    {
+        if ($this->loginConfigBuilder instanceof LoginConfigBuilder) {
```

Comment:
Why is this builder nullable? 🤔

Reply:
I do it, because there were issues while updating Shopware. (Pipeline)

`Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException {#10714
  #message: "The service "Shopware\Core\Framework\Api\Controller\AuthController" has a dependency on a non-existent service "Shopware\Core\LoginConfig\Builder\LoginConfigBuilder"."`